### DISCUSSION TITLE
Using database encrypted

### DISCUSSION THREAD

**Discussion by danilobatistaqueiroz** [capacitor-community/sqlite#427] [Q&A] [answered]
How can I use the method `setEncryptionSecret(String passphrase)` ?
Because anyone can decompile or inspect all javascript files inside the aab or apk finding my passphrase.  
Is there a way to obfuscate or to make the job to find the passphrase very hard?

**Comment by jepiqueau** [comment, marked as answer]
@danilobatistaqueiroz the passphrase is stored in SharedPreferences. Not in javascript files
